src/renderer/navigation/history.ts

```diff
@@ -1,6 +1,15 @@
 import { ipcRenderer } from "electron";
 import { createBrowserHistory, createMemoryHistory } from "history";
 import { createObservableHistory } from "mobx-observable-history";
+import { getHostedClusterId } from "../../common/cluster-store";
+import logger from "../../main/logger";
```

Comment:
unrelated, but shouldn't `logger` be in `src/common`?

Reply:
Yes it probably should be. I too have thought about that. Lets see about doing that for 4.3